.github/workflows/terragrunt-apply-production.yml

```diff
@@ -190,12 +190,11 @@ jobs:
 
   update-lambda-function-image:
     needs: [detect-lambda-changes, terragrunt-apply-all-modules]
-    if: needs.detect-lambda-changes.outputs.lambda-to-rebuild != '[]'
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        image: ${{ fromJSON(needs.detect-lambda-changes.outputs.lambda-to-rebuild) }}
+        image: [audit-logs, audit-logs-archiver, cognito-email-sender, cognito-pre-sign-up, form-archiver, load-testing, nagware, notify-slack, reliability, reliability-dlq-consumer, response-archiver, submission, vault-integrity]
```

Comment:
Could you define this static array as an environment variable and then use it for both spots.

I'd say we can also remove the `detect-lambda-changes` and all its reference since this isn't doing anything now.

Reply:
Yes absolutely! 
I went with this approach because I want to revisit that in the future as I would like to still be able to use the `lambda-filter.yml` file to dynamically create an array so that there is only one place where we add / remove lambdas